crates/ruff_linter/src/rules/ruff/rules/mod.rs

```diff
@@ -45,6 +45,8 @@ mod parenthesize_logical_operators;
 mod sequence_sorting;
 mod sort_dunder_all;
 mod sort_dunder_slots;
+pub(crate) use static_key_dict_comprehension::*;
```

Comment:
(I think this should be in the section above.)

Reply:
Was fixing :)